### PR TITLE
feat(query): Added Service Account Key File Not Properly Set for Kubernetes

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- Added Service Account Key File Not Properly Set for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/service_account_key_file_not_properly_set/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "dab4ec72-ce2e-4732-b7c3-1757dcce01a1",
+  "queryName": "Service Account Key File Not Properly Set",
+  "severity": "MEDIUM",
+  "category": "Secret Management",
+  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be difined and set a PEM encoded file",
```
Comment (joaorufi) [inline assets/queries/k8s/service_account_key_file_not_properly_set/metadata.json:6]: Misspelling the word defined
```suggestion
  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be defined and set a PEM encoded file",
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/service_account_key_file_not_properly_set/query.rego [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--service-account-key-file flag should have a PEM encoded file",
+		"keyActualValue": "--service-account-key-file flag is not a PEM encoded file",
+		"searchLine": common_lib.build_search_line(split(specInfo.path, "."), [types[x], j, "command"]),
+	}
+}
```
Comment (rafaela-soares) [inline assets/queries/k8s/service_account_key_file_not_properly_set/query.rego:24]:
```suggestion
```
I think this policy is unnecessary

File: assets/queries/k8s/service_account_key_file_not_properly_set/metadata.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "dab4ec72-ce2e-4732-b7c3-1757dcce01a1",
+  "queryName": "Service Account Key File Not Properly Set",
+  "severity": "MEDIUM",
+  "category": "Secret Management",
+  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be defined and set a PEM encoded file",
```
Comment (rafaela-soares) [inline assets/queries/k8s/service_account_key_file_not_properly_set/metadata.json:6]:
```suggestion
  "descriptionText": "When using kube-apiserver command, the '--service-account-key-file' flag should be defined",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Review by joaorufi — DISMISSED** [review]

**Pull request merged**